.github/composite_actions/run_xcodebuild_test/action.yml

```
@@ -0,0 +1,32 @@
+name: 'Run Test'
+description: 'Action runs the test specified at project_path for workspace and scheme'
+
+inputs:
+  project_path:
+    required: true
+    type: string
+  workspace:
+    required: true
+    type: string
+  scheme:
+    required: true
+    type: string
+  simulator:
+    required: false
+    type: string
+    default: 'platform=iOS Simulator,name=iPhone 13,OS=15.0'
```

```suggestion
    default: 'platform=iOS Simulator,name=iPhone 13,OS=latest'
```
I ran into an issue while adding unit tests where whatever OS version I specified (it may have been 15.0) wasn't available.  I solved it by changing the version to `latest` which I think is a good default anyway so that we are always running tests against the latest version.